bcl2fastq/index_summary.py

```diff
@@ -4,15 +4,16 @@
 import dash_html_components as html
 import dash.dependencies as dep
 import pandas
+import math
 
-index = gsiqcetl.bcl2fastq.parse.load_cache(
+index1 = gsiqcetl.bcl2fastq.parse.load_cache(
```

Comment:
I wouldn't rename this to `index1` as "Index1" already has a very specific meaning in the rest of the script ("the label for the Index 1 column") that doesn't correspond to this variable, which holds data for all the runs.
  (`index` is already an ambiguous name because of this, but changing it is beyond the scope of this issue)
Does the variable need to be renamed for line 16 to work, or can you keep using the same pattern as elsewhere in the file, like `index = index.dropna(...)`?

Reply:
This variable needed to be renamed for a downstream function.  Line 177 requires a pure version of the variable to correctly display the pie graph. But I will rename it to something not already in use